### PR TITLE
Add Meteor register and login

### PR DESCRIPTION
**Pull request by xasopheno**
I got the regular Meteor login working.

### CODE REVIEW

**Comment by lpender** [comment]
Could you add a screenshot to this?

Or better yet, you could make a little GIF of you logging in, using http://www.cockos.com/licecap/ !

File: .meteor/packages [hunk 1]
```diff
@@ -19,3 +19,8 @@ ecmascript@0.5.7              # Enable ECMAScript2015+ syntax in app code
 
 autopublish@1.0.7             # Publish all data to the clients (for prototyping)
 insecure@1.0.7                # Allow all DB writes from clients (for prototyping)
+
+meteortoys:allthings
```
Comment (lpender) [inline .meteor/packages]: What does `meteortoys:allthings` do for us?

File: .meteor/packages [hunk 2]
```diff
@@ -19,3 +19,8 @@ ecmascript@0.5.7              # Enable ECMAScript2015+ syntax in app code
...
+kadira:flow-router			  # Router for Meteor
+accounts-ui
+accounts-password
```
Comment (lpender) [inline .meteor/packages]: This little red "uh uh" symbol means that there's no new line at the end of the document. For some reason, it's preferred that all files end with a carriage return.

You can configure sublime to do this (somehow) 

https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842

File: client/AccountsUI.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+
+export default class AccountsUI extends Component {
```
Comment (lpender) [inline client/AccountsUI.jsx]: Nice! This is \es6 ?

File: client/routes.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { mount } from 'react-mounter'; 
+
+import { MainLayout } from './layouts/MainLayout.jsx';
+
+FlowRouter.route('/', {
+	action() {
```
Comment (lpender) [inline client/routes.jsx]: Can we go with two spaces for indentation?

File: package.json [hunk 5]
```diff
@@ -1,10 +1,20 @@
...
   "private": true,
   "scripts": {
     "start": "meteor run"
   },
   "dependencies": {
-    "meteor-node-stubs": "~0.2.0"
+    "meteor-node-stubs": "~0.2.0",
+    "react": "^15.3.0",
+    "react-dom": "^15.3.0",
+    "react-mounter": "^1.2.0"
```
Comment (lpender) [inline package.json]: Sweet. I think I agree the right way to do this is probably through NPM, not some Atmosphere package.

**Comment by lpender** [comment]
👍 LGTM

**Comment by xasopheno** [comment]
​Learned to make the gif. Great tool. Thanks. Danny

Danny Meyer
We can do great things if we do them together.
US +1 (970)-420-7526
Skype: xasopheno
www.ThisIsAmazing.org
<http://www.thisisamazing.org/>www.TheDJMLC.com <http://www.thedjmlc.com/>

**Pull request merged**